javascript/script.js

```diff
@@ -0,0 +1,65 @@
+const preLoad = document.querySelector("#loading");
+const sideBar  = document.querySelector(".sidebar") 
+const Input_validate = document.querySelector(".ip-validate") 
```

Comment:
You should keep the var names in snakeCase. Caps are treated as class names by IDEs, so the color is changed also.